### DISCUSSION TITLE
(Fixed) Do not upgrade bluez-utils to 5.65

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#980] [General]
Upgrading `bluez-utils` to `5.65` breaks Bluetooth functions, missing `paired-devices` command.

**Comment by klassikmann** [comment]
Sigh! Too late ...

**Reply by rern** [reply]
Roll back:
```sh
wget https://alaa.ad24.cz/packages/b/bluez-utils/bluez-utils-5.64-2-aarch64.pkg.tar.xz
pacman -U bluez-utils-5.64-2-aarch64.pkg.tar.xz
```

**Reply by klassikmann** [reply]
> Roll back:

Done. Plus adding bluez-utils as IgnorePkg in /etc/pacman.conf

Thank you

**Reply by cdevalland** [reply]
Hello,
no more bluetooth with version 20220904
I tried this roll back but still not work.

**Reply by cdevalland** [reply]
If it can help :
`rAudio:~  #  systemctl status bluetooth
* bluetooth.service - Bluetooth service
     Loaded: loaded (/usr/lib/systemd/system/bluetooth.service; enabled; preset: disabled)
    Drop-In: /etc/systemd/system/bluetooth.service.d
             `-override.conf
     Active: inactive (dead)
  Condition: start condition failed at Mon 2022-09-05 10:37:54 CEST; 52s ago
             `- ConditionPathIsDirectory=/sys/class/bluetooth was not met
       Docs: man:bluetoothd(8)

Sep 05 10:37:38 rAudio systemd[1]: Bluetooth service was skipped because of a failed condition check (ConditionPathIsDirectory=/sys/class/bluetooth).
Sep 05 10:37:54 rAudio systemd[1]: Bluetooth service was skipped because of a failed condition check (ConditionPathIsDirectory=/sys/class/bluetooth).
`

**Comment by rern** [comment]
With update 20220814, this has been fixed by avoiding the missing command.

**Reply by klassikmann** [reply]
Thank you. 20220814 deleted bluez-utils as IgnorePkg in /etc/pacman.conf.

**Comment by cdevalland** [comment]
Hi,
no more bluetooth even after reboot (cursor always off) with update 20220814.
Can I try something ?
Thanks for help.
